packages/cli/src/scaling/scaling.types.ts

```diff
@@ -28,16 +28,13 @@ export type JobStatus = Bull.JobStatus;
 
 export type JobOptions = Bull.JobOptions;
 
-/** Message sent by worker to queue or by queue to worker. */
-export type JobMessage = RepondToWebhookMessage | AbortJobMessage;
-
-export type RepondToWebhookMessage = {
+export type MessageToMain = {
 	kind: 'respond-to-webhook';
 	executionId: string;
 	response: IExecuteResponsePromiseData;
 };
 
-export type AbortJobMessage = {
+export type MessageToWorker = {
```

Comment:
I think these would be clearer if the types were still called `RespondToWebhookMessage` and `AbortJobMessage`, as that's what they semantically are. If we want we can then create aliaes for `MessagesToMain` and `MessagesToWorker` which alias these types